### ISSUE TITLE
[Container Cert] Send Container Certification Results to Pyxis

### ISSUE THREAD

**Issue by tonytcampbell** [redhat-openshift-ecosystem/openshift-preflight#369]
`preflight check container` command should include an option (maybe a flag) to `--submit` results.  When this options is selected the results should be submitted to the Pyxis API.

**Comment by amisstea** [comment]
@jomkz we just added https://issues.redhat.com/browse/ISV-1664 to our backlog with the expectation that preflight could "phone home" the Started -> In Progress certification state transition. It could happen at any time the partner first starts working with preflight but it could also occur during submission so this seems to be the most relevant issue. Please take a look and let me know if you have any questions or concerns. Thanks!

**Comment by acornett21** [comment]
@amisstea Should we do a call to `GET` call to `projects/certification/id/1234` to verify the status is in a status of `Started` before we call `PATCH` with the new `In Progress` status?

**Comment by amisstea** [comment]
@acornett21 yes, that is probably a good idea. We may allow it to be `PATCH`'d to `In Progress` if currently in that state but going from `Published` to `In Progress` is expected to return an error.
